File: algorithms/search/linearSearch/f_linearSearch.js
const { performance } = require("perf_hooks");

const main = () => {
  const unsortedNumbers = [21, 23, 4, 3, 8];
  const target = 80;
  let found = false;

  const timerStart = performance.now();

  for (let i = 0; i < unsortedNumbers.length; i++) {
    if (unsortedNumbers[i] == target) {
      found = true;
    }
  }

  const timerEnd = performance.now();

  console.log(found);
  console.log(`Elapsed time: ${timerEnd - timerStart}`);
};

main();

File: algorithms/search/linearSearch/f_linearSearch.py
import time

def main(): 
  unsortedNumbers = [5, 23, 80, 32, 5, 1]
  target = 18
  found = False

  timerStart = time.time()

  for number in unsortedNumbers:
    if number == target:
      found = True

  timerEnd = time.time()
  
  print(found)
  print(f'Elapsed time: {timerEnd - timerStart}')

main()